src/reflection.jl

```diff
@@ -130,20 +132,21 @@ function process_info(interp, @nospecialize(info), argtypes::ArgTypes, @nospecia
         infos = process_recursive(info.call)
         @assert length(infos) == length(info.results)
         return mapany(enumerate(info.results)) do (i, result)
-            if isnothing(result)
+            if isnothing(result) || isa(result, Bool)
```

Comment:
I'm curious, what does a `Bool` here mean?

Reply:
Not 100% sure, I left it in while rebasing from https://github.com/JuliaDebug/Cthulhu.jl/pull/266/files#diff-11ccfcf41873e6a1cf01d00ed11aee2a59b7c1c1634cfa80c84b8049c6c34564R137-R143

Reply:
Bool was dropped from the final version of the base change, so should be dropped here also.